docs/CHANGELOG.rst

```diff
@@ -23,6 +23,25 @@ Fixed
 Security
 ~~~~~~~~
 
+[2.7.0] - 2020-08-04
+--------------------
+
+Added
+~~~~~
+- Add encoding support to Path / HDFSFile
+- Movielens example
+
+Changed
+~~~~~~~
```

Comment:
- Changed default initializer for embeddings in the embeddings layer

Reply:
done